src/ducks/context/BanksContext.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { createContext, useContext, useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Q } from 'cozy-client'
+import { KONNECTOR_DOCTYPE } from 'doctypes'
+import { useJobsContext } from 'ducks/context/JobsContext'
+
+export const BanksContext = createContext({})
+
+export const useBanksContext = () => {
+  return useContext(BanksContext)
+}
+
+/**
+ * BanksProvider
+ *
+ * @param children
+ * @param client
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const BanksProvider = ({ children, client }) => {
+  const { jobsInProgress = [] } = useJobsContext()
+  const [banksJobsInProgress, setBanksJobsInProgress] = useState([])
+
+  useEffect(() => {
+    const arr = [...jobsInProgress]
+    if (arr.length > 0) {
+      const formatJobsInProgress = async () => {
+        const ids = arr.map(jobsInProgress => {
+          const slug = jobsInProgress.konnector
+          return `${KONNECTOR_DOCTYPE}/${slug}`
+        })
+        const resp = await client.query(Q(KONNECTOR_DOCTYPE).getByIds(ids))
+        const newJobInProgress = resp.data.map(konnector => {
+          const slug = konnector.slug
+          const jobInProgress = jobsInProgress.find(j => j.konnector === slug)
+          return {
+            konnector: slug,
+            account: jobInProgress.account,
+            institutionLabel: konnector.name
+          }
```

Comment:
Nit: I am not sure this is necessary to "commit" to a format so early. This format could be done in the view, and the provider could here only return the konnector/account/institutionLabel.